### PR TITLE
Issue #2366: Added LoopProtection to GetFieldStates POD.

### PR DESCRIPTION
**Pull request by stefanhaerter**
Fixing #2366

### CODE REVIEW

**Review by bschmalhofer — DISMISSED** [review]
Documenting the current state is good. Approved.

File: Kernel/System/Ticket/FieldRestrictions.pm [hunk 1]
```diff
@@ -80,9 +80,11 @@ Returns possible values, selected values, and visibility of fields
             %GetParam,
             OwnerID     => $GetParam{NewUserID},
         },
+        LoopProtection      => 100,                                 # restricts number of recursive calls; passing 'undef' will lead to a warning
```
Comment (bschmalhofer) [inline Kernel/System/Ticket/FieldRestrictions.pm]: Can LoopProtection really be 100? My understandig is that a reference to a scalar must be passed.

Comment (stefanhaerter) [reply Kernel/System/Ticket/FieldRestrictions.pm]: You are right, this has to be a scalar reference. I will adjust the POD accordingly - thank you! =)

**Review by bschmalhofer — APPROVED** [review]
Yes, this emphasises that the loop protection variable will be modified within the method.

**Comment by bschmalhofer** [comment]
Moved this to rel-11_0, as adding those minor improvements to maintainance branches is not worth the hassle.

**Pull request merged**